### ISSUE TITLE
Add a screen reader-only "skip nav" link

### ISSUE THREAD

**Issue by drewbrew** [hsv-dot-beer/hsvdotbeer-nuxt#56]
More accessibility improvements

> Basically, first element of body is a screen reader only (sr-only in bootstrap unless it's changed) hyperlink that when clicked goes to the start of the page content and doesn't read the navigation/logo sections
>
> Though it's more optional on your site because it's just one page, it's mostly meant for skipping content that's just repeated on every page of a site
